service/history/handler.go

```diff
@@ -1295,6 +1295,44 @@ func (h *Handler) ReplicateRawEvents(
 	return nil
 }
 
+// ReplicateEventsV2 is called by processor to replicate history events for passive domains
+func (h *Handler) ReplicateEventsV2(
+	ctx context.Context,
+	replicateRequest *hist.ReplicateEventsV2Request,
+) (retError error) {
```

Comment:
Looks like we never use retError?

Reply:
the first line within this API use it
`defer log.CapturePanic(h.GetLogger(), &retError)`